ReactWindows/ReactNative.Shared/UIManager/Events/KeyHelpers.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+#if WINDOWS_UWP
+using Windows.System;
+#else
+using System.Windows.Input;
+#endif
+
+namespace ReactNative.UIManager.Events
+{
+#if WINDOWS_UWP
+    using Key = VirtualKey;
+#endif
+
+    static class KeyHelpers
+    {
+        public static int GetKeyCode(this Key key)
```

Comment:
>GetKeyCode [](start = 26, length = 10)

Is this still used?

Reply:
Yes, it's used to retrieve the numerical value to pass through the bridge.

---
In reply to: [157096384](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r157096384) [](ancestors = 157096384)